### ISSUE TITLE
Icon request: Serial Port

### ISSUE THREAD

**Issue by mexicocss** [google/material-design-icons#1634]
A serial port icon would be great to be include on it thank you so much.

**Comment by tphinney** [comment]
Really? Like an old 9-pin RS-232 port?

I gather they are still used in data acquisition devices, but they have not been much built into computers for many years now.

**Comment by mexicocss** [comment]
yes Thomas i still using this specially in industrial development to connect to scales, thermometers and other devices that are still using db9 connections, please consider my request :)
